scripts/collections-sql/hurricane_ida_blue_tarps.sql

```diff
@@ -0,0 +1,32 @@
+INSERT INTO pgstac.collections (content) VALUES('{
+   "id": "Hurricane_Ida_Blue_Tarps",
+   "type": "Collection",
+   "links":[
+   ],
+   "title":"Hurricane Ida Blue Tarps",
+   "extent":{
+      "spatial":{
+         "bbox":[
+            [
+               -90.300691019583,
+               29.791754950316868,
+               -89.86300184384689,
+               30.099979027371006
+            ]
+         ]
+      },
+      "temporal":{
+         "interval":[
+            [
+               "2021-09-26T00:00:00Z",
+               "2021-11-25T00:00:00Z"
```

Comment:
The collection contains files for two dates only: Sept 26, 2021 and Nov 24, 2021.
Is this the right way to do this? @abarciauskas-bgse

Reply:
Sounds good to me